packages/components/src/animate/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+function Animate( { type, options = {}, children } ) {
```

Comment:
This component should be documented on its own to ensure developers can use it to animate their own components. To be honest, I'm not quite sure how one could use it inside Gutenberg, too.